packages/test-utils/src/getAccessToken.ts

```diff
@@ -1,5 +1,5 @@
 import { ReplayReporterConfig } from "./types";
 
-export function getAccessToken(config: ReplayReporterConfig<any>) {
-  return config.apiKey || process.env.REPLAY_API_KEY || process.env.RECORD_REPLAY_API_KEY;
+export function getAccessToken(config?: ReplayReporterConfig<any>) {
```

Comment:
It could be good to include here similar logs as we have in `packages/shared/src/authentication/getAccessToken.ts`